crates/cashu/src/nuts/nut02.rs

```diff
@@ -45,6 +45,15 @@ pub struct Id {
 
 impl Id {
     const STRLEN: usize = 14;
+
+    pub fn to_bytes(&self) -> Vec<u8> {
+        hex::decode(self.to_string()).unwrap()
```

Comment:
i would rather we change the function signature to a result

Reply:
then maybe the function name should be `try_into_bytes` :thinking:

Reply:
I don't think we should change the signature because this _should_ never fail. However, the only reason it can fail now is that we store the hex string instead of the bytes in this object. Ideally, the `Id` struct should hold the bytes. Should I mark this as a TODO for now and follow up with a refactor shortly?

Reply:
Yes that makes sense